src/modules/form/components/group/TableGroup.tsx

```diff
@@ -131,25 +156,7 @@ const TableGroup = ({
             ))}
           </TableRow>
         </TableHead>
-        <TableBody>
-          {item.item?.map((rowItem) => (
-            <TableRow key={rowItem.linkId}>
-              {rowItem.item?.map((cellItem, index) => (
-                <TableCell key={cellItem.linkId}>
-                  {renderChildItem(
-                    // remove helper text, it is shown in table header
-                    { ...cellItem, helperText: null },
```

Comment:
this was the line causing the problem, stabilizing cellItem using memoizing seems to fix the issue

Reply:
Nice find, thank you!
The intention I believe is to hide the Helper Text in the table cells, since they are pulled out and displayed in the table row. It looks like that behavior got lost with your change, I added it back https://github.com/greenriver/hmis-frontend/pull/1091/commits/b25d638d71fc5845debe2bf87408cf53d2907e9c using a deep-clone approach to keep a copy of the modified `item`s. It seems to work.



before (Storybook):
<img width="1161" alt="Screenshot 2025-03-07 at 7 55 19 AM" src="https://github.com/user-attachments/assets/291795e9-0583-40c5-ac17-6377d761e40f" />

after: (back to original behavior)
<img width="1160" alt="Screenshot 2025-03-07 at 7 52 20 AM" src="https://github.com/user-attachments/assets/aef97279-0da1-4eb2-b07f-16a93407cbd1" />

Reply:
> The intention I believe is to hide the Helper Text in the table cells, since they are pulled out and displayed in the table row

Ah oops, sorry about that, thanks for fixing!